pkg/volume/util/util.go

```diff
@@ -721,3 +722,21 @@ func CheckVolumeModeFilesystem(volumeSpec *volume.Spec) (bool, error) {
 	}
 	return true, nil
 }
+
+// MakeAbsolutePath convert path to absolute path according to GOOS
+func MakeAbsolutePath(goos, path string) string {
+	if goos != "windows" {
+		return "/" + filepath.Clean(path)
```

Comment:
Clean should be called after prepending "/".  Ie, if the user already specified an absolute path.  A test case should be added too

Reply:
@msau42 thanks, fixed, and added a new unit test